diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,45 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request_uri, request_uri_mode=POST)
+deactivate r
+u --> w: authorization request\n(client_id, request_uri, request_uri_mode=POST)
+activate w
+w --> rp: POST **request_uri** (\n[OPTIONAL]issuer_metadata, \n[OPTIONAL]wallet_nonce)
```

Comment:
```suggestion
w --> rp: POST **request_uri** (\n[OPTIONAL]wallet_metadata, \n[OPTIONAL]wallet_nonce)
```

`wallet_metadata` as  defined in https://github.com/openid/OpenID4VP/pull/59/files#diff-3118c9756a1d7b361bb53af8bd9d65666476a27cfeeced56c47a5dccc38eac55R488